pkgs/by-name/ty/typos-lsp/package.nix

```
@@ -0,0 +1,26 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "typos-lsp";
+  version = "0.1.12";
+
+  src = fetchFromGitHub {
+    owner = "tekumara";
+    repo = pname;
+    rev = "v${version}";
```

```suggestion
    rev = "refs/tags/v${version}";
```
This is recommended to avoid (unlikely but) eventual tag-branch confusion.